packages/firestore/src/api/user_data_reader.ts

```diff
@@ -476,6 +446,21 @@ export class UserDataReader {
     );
   }
 
+  /** Creates a new top-level parse context. */
+  private createContext(
+    dataSource: UserDataSource,
+    methodName: string
+  ): ParseContext {
+    return new ParseContext({
+      dataSource,
+      databaseId: this.databaseId,
+      serializer: this.serializer,
+      methodName,
```

Comment:
There is a lint setting in our repo that prohibits non-shorthand property notations if they are possible. 
E.g. `{ foo1 : foo }` is allowed but since `{ foo: foo }` is the same as `{ foo }`  we are meant to use `{ foo }`

See also https://alligator.io/js/object-property-shorthand-es6/

Reply:
TIL about "Object Property Value Shorthand"!